web/webapp/copy-env.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env sh
+set -eux
+
+# copy .env.local.example to .env if .env doesn't already exist
+if [ ! -f .env ]; then
+    cp .env.local.example .env
+    echo ".env file created from .env.local.example"
+fi
+
+sleep 2
```

Comment:
Why is this sleep needed?

If you ware worried about fine not being actually copied yet, I would rather use `sync .env` (https://phoenixnap.com/kb/linux-sync#ftoc-heading-5 - this is forcing the kernel to actually write the file from cache to the disk)